.env.example

```
@@ -22,6 +22,7 @@ SQUIRE_ALLOWED_EMAILS=your-email@example.com
 LANGFUSE_PUBLIC_KEY=
 LANGFUSE_SECRET_KEY=
 LANGFUSE_BASEURL=https://us.cloud.langfuse.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LANGFUSE_BASEURL key should go before the LANGFUSE_PUBLIC_KEY key

(UnorderedKey)